src/callbacks_step/save_solution.jl

```diff
@@ -6,23 +6,26 @@
 
 
 """
-    SaveSolutionCallback(; interval=0,
+    SaveSolutionCallback(; interval::Integer=0, dt=0.0,
                            save_initial_solution=true,
                            save_final_solution=true,
                            output_directory="out",
                            solution_variables=cons2prim)
 
-Save the current numerical solution every `interval` time steps. `solution_variables` can be any
-callable that converts the conservative variables at a single point to a set of solution variables.
-The first parameter passed to `solution_variables` will be the set of conservative variables and the
-second parameter is the equation struct.
+Save the current numerical solution in regular intervals. Either pass `interval` to save
+every `interval` time steps, or pass `dt` to save in intervals of `dt` in terms
+of integration time by adding additional `tstops` (note that this may change the solution).
```

Comment:
@sloede We discussed whether we want to do this or use some time that is close enough, I think. Do you remember the outcome?

Reply:
No, I don't remember. Although I am thinking now whether we should really rely on DiffEqCallbacks or if we should implement it ourselves, since the current implementation won't work for AMR meshes, will it?

Reply:
Why wouldn't it work for AMR meshes?

Reply:
Ah, now I am confused. So this is *not* interpolating but actually adding additional time steps? Maybe I got this wrong from looking at this cursorily

Reply:
Yes, it's adding additional `tstops` for exactly this reason.